left over SetSysClock declaration for NAZE target (harmless irritation)

The following was missed in an earlier cleanup:

```
./src/main/drivers/system_stm32f10x.c:31:6: warning: type of 'SetSysClock' does not match original declaration [-Wlto-type-mismatch]
 void SetSysClock(bool overclock);
      ^
./lib/main/CMSIS/CM3/DeviceSupport/ST/STM32F10x/system_stm32f10x.c:90:6: note: type mismatch in parameter 1
 void SetSysClock(void)
      ^
./lib/main/CMSIS/CM3/DeviceSupport/ST/STM32F10x/system_stm32f10x.c:90:6: note: type 'void' should match type '_Bool'
./lib/main/CMSIS/CM3/DeviceSupport/ST/STM32F10x/system_stm32f10x.c:90:6: note: 'SetSysClock' was previously declared here
```

--- a/src/main/drivers/system_stm32f10x.c
+++ b/src/main/drivers/system_stm32f10x.c
@@ -28,7 +28,7 @@
 #define AIRCR_VECTKEY_MASK    ((uint32_t)0x05FA0000)
 
 // from system_stm32f10x.c
-void SetSysClock(bool overclock);
+void SetSysClock(void);
 
 void systemReset(void)
 {
@@ -70,7 +70,7 @@ void systemInit(void)
 {
     checkForBootLoaderRequest();
 
-    SetSysClock(false);
+    SetSysClock();
 
 #ifdef CC3D
     /* Accounts for OP Bootloader, set the Vector Table base address as specified in .ld file */

